.eslintrc.js

```diff
@@ -22,16 +22,18 @@ module.exports = {
         // ! we have changed the configuration rules from error to warning for now to avoid crashing the deploy.
         // ! This does not mean the below rules are meant to stay as if, the warning must be fixed until
         // ! linter does not complain about a specific rule and can be safely removed from below.
-        '@typescript-eslint/no-unsafe-assignment': 'warn',
```

Comment:
Some rules has been changed to 'off' as eslint is so strict to start migrating the app. Comment has been added